BFit/Controller/ProfileViewController.swift

```diff
@@ -57,8 +59,7 @@ class ProfileViewController: UIViewController, UIImagePickerControllerDelegate,
             response in
             if response.result.isSuccess {
                 self.postsData = JSON(response.data!)["posts"]
-                print(self.postsData)
-//                self.postsTable.performSelector(onMainThread: #selector(UICollectionView.reloadData), with: nil, waitUntilDone: true)
+                self.postsTable.performSelector(onMainThread: #selector(UICollectionView.reloadData), with: nil, waitUntilDone: true)
```

Comment:
this "waitUntilDone" I'm assuming is swift's version of async?

Reply:
Yeah, it's one way of doing it I believe